app/models/edition.rb

```diff
@@ -195,6 +195,12 @@ def self.to_date(date)
     where("editions.updated_at <= ?", date)
   end
 
+  # used by Admin::EditionFilter
+  def self.only_broken_links
+    joins("INNER JOIN link_checker_api_reports ON link_checker_api_reports.link_reportable_id = editions.id")
+    .joins("INNER JOIN link_checker_api_report_links ON link_checker_api_report_links.link_checker_api_report_id = link_checker_api_reports.id")
```

Comment:
Should one of these joins be checking that there is a status of broken?

Reply:
Correct, I've added a `where` clause to catch this